docs/source/community/communication.md

```diff
@@ -54,11 +54,11 @@ can also ask for one in our Discord server.
 
 ### Sync up Zoom calls
 
-We have biweekly sync calls every other Thursdays at 16:00 UTC
-(starting September 30, 2021) on Zoom [Meeting Link](https://influxdata.zoom.us/j/94666921249)
+We have biweekly sync calls every other Thursdays at 04:00 UTC and 16:00 UTC
```

Comment:
So is it 04:00 UTC or 16:00 UTC?

Reply:
I think @bkmgit  was running one at 04:00UTC  -- though I am not sure if that is still happening

Reply:
Yes, will continue with this. There have been promising possibilities of presentations at this time.